.gitignore

```diff
@@ -35,3 +35,5 @@ resinrc.yml
 
 /tmp
 build/
+build-bin/
+resin-cli-*.zip
```

Comment:
NIT: would it make sense emit the zip files ain a `build-zip` folder to keep things tidy.

Reply:
Good idea, done